ui.frontend/src/view/FormFieldBase.js

```diff
@@ -176,6 +177,7 @@ class FormFieldBase extends FormField {
         let errorElement = typeof this.getErrorDiv === 'function' ? this.getErrorDiv() : null;
         if (errorElement) {
             errorElement.setAttribute('id', `${this.getId()}__errormessage`);
+            errorElement.setAttribute('aria-describedby', `${this.getId()}__label`);
```

Comment:
It is already present in the input field. We can not use the aria-describedby attribute on the error container.